BUILDING.md

```diff
@@ -29,3 +29,18 @@ To build with only the TinkerPop tests:
 ```
 mvn clean install -Dtest.skip.tp=false -DskipTests=true
 ```
+
+## Building on Eclipse IDE
+Note that this has only been tested on Eclipse Neon.2 Release (4.6.2) with m2e (1.7.0.20160603-1933) and m2e-wtp (1.3.1.20160831-1005) plugin.
+
+To build without executing tests:
+1. Right-click on your project -> "Run As..." -> "Run Configurations..."
```

Comment:
Insert a blank line above this line; otherwise, it doesn't end up as a list (see your [rendered README](https://github.com/blacknred0/janusgraph/blob/ad51e0a642d43fc5dce8b11efac55909c8d22579/BUILDING.md) on your branch).